### ISSUE TITLE
[todo] Create an issue on csswg-drafts regarding adding a function to the 'text-indent' property that ignores full-width spaces

### ISSUE THREAD

**Issue by kidayasuo** [w3c/jlreq#423]
This is outcome from the [F2F meeting on 2024-4-17](https://github.com/w3c/jlreq/discussions/421#discussioncomment-9160838).

---

### [4. White Space Processing & Control Characters](https://www.w3.org/TR/css-text-4/#white-space-processing)

In Japanese text, the style of indenting the first character of a paragraph is usually achieved by inserting a full-width space. To change this paragraph style, it's necessary to ignore this full-width space character at the start. A property that enables this is required.

After reviewing several approaches with Fantasai, Mr. Murakami's proposal to add a function to the 'text-indent' property that ignores full-width spaces was deemed good. This is similar to the 'replace' function in text-autospace that ignores spaces between Japanese and Western text.

##### Actions
Mr. Murakami will create an issue on [csswg-drafts](https://github.com/w3c/csswg-drafts/labels/css-text-4) regarding this proposal.

**Comment by xfq** [comment]
Indeed, and a similar situation exists in Chinese: https://www.w3.org/TR/clreq/#h-note-36
